lib/mongoid/persistable/maxable.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Mongoid
+  module Persistable
+
+    # Defines behavior for setting a field (or fields) to the larger of
+    # either it's current value, or a given value.
+    module Maxable
```

Comment:
General comment but I don't see how having one module per method makes the code more readable. Probably all these Persistable atomic-like methods should be grouped into a single module.

Reply:
I actually disagree here; looking at the `Contextual::Atomic` module, I've considered breaking it up so that single module isn't so monolithic. (I won't, though---other priorities!) A compromise could be to group similar methods (e.g. `min` and `max`) into modules. Something to consider for future work.

Reply:
It's a matter of taste. At my company, we only use mixins where there is a bona-fide need to add the same method(s) to two or more classes. When I see a mixin like `Maxable` or `Datable`, firstly the names are very silly, but a bigger problem is that it's not obvious to me what other classes/modules besides `Persistable` might be including `Maxable` and `Datable`, and what knock-on effects refactoring them might cause.